<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatherPad::PrefDialog</class>
 <widget class="QDialog" name="prefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Window</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_1">
       <item row="0" column="0">
        <widget class="QScrollArea" name="windowScrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents1">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>434</width>
            <height>442</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="windowGoupBox">
             <property name="title">
              <string>Window Settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_1">
              <item>
               <widget class="QCheckBox" name="winSizeBox">
                <property name="text">
                 <string>Remember window &amp;size on closing</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="sizeLable">
                  <property name="toolTip">
                   <string>Window frame is excluded.</string>
                  </property>
                  <property name="text">
                   <string>Start with this size: </string>
                  </property>
                  <property name="indent">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinX">
                  <property name="toolTip">
                   <string>Window frame is excluded.</string>
                  </property>
                  <property name="suffix">
                   <string> px</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="mLabel">
                  <property name="toolTip">
                   <string>Window frame is excluded.</string>
                  </property>
                  <property name="text">
                   <string>Ã—</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinY">
                  <property name="toolTip">
                   <string>Window frame is excluded.</string>
                  </property>
                  <property name="suffix">
                   <string> px</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::MinimumExpanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>5</width>
                    <height>5</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="winPosBox">
                <property name="toolTip">
                 <string>Enforcing a window position is not recommended
and may not work with some window managers.</string>
                </property>
                <property name="text">
                 <string>Remember window &amp;position on closing</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_1">
                <item>
                 <spacer name="horizontalSpacer_1">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>22</width>
                    <height>5</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="searchHistoryBox">
                <property name="toolTip">
                 <string>By default, each search entry
has a separate search history.</string>
                </property>
                <property name="text">
                 <string>Use a shared search history</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="statusBox">
                <property name="text">
                 <string>Always show st&amp;atus bar</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>22</width>
                    <height>5</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="statusCursorsBox">
                  <property name="text">
                   <string>Show cursor position on status bar</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="tabLabel">
                  <property name="toolTip">
                   <string>Will take effect after closing this dialog.</string>
                  </property>
                  <property name="text">
                   <string>Tab position: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="tabCombo">
                  <property name="toolTip">
                   <string>Will take effect after closing this dialog.</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>North</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>South</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>West</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>East</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::MinimumExpanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>5</width>
                    <height>5</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="tabBox">
                <property name="toolTip">
                 <string>This means that, for LTR, Alt+Right goes to the first tab
after the last tab is activated, and the same for Alt+Left.

Tab navigation with mouse wheel is not affected.</string>
                </property>
                <property name="text">
                 <string>Tab navigation wraps &amp;around</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::MinimumExpanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>5</width>
                  <height>1</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QScrollArea" name="textScrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>393</width>
            <height>522</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="textGoupBox">
             <property name="title">
              <string>Text Editor</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="textTabLabel">
                  <property name="text">
                   <string>Text tab size:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="textTabSpin">
                  <property name="suffix">
                   <string> spaces</string>
                  </property>
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>10</number>
                  </property>
                  <property name="value">
                   <number>4</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Files</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>File Management</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="unmodifiedSaveBox">
            <property name="toolTip">
             <string>If this is checked, the Save button/menu-item could
be used also when the opened file is not modified.

This can be useful under certain circumstances,
e.g. for changing the time stamp or removing the
trailing spaces of a text file.</string>
            </property>
            <property name="text">
             <string>Allow saving unmodified files</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Shortcuts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Action</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Shortcut</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="defaultButton">
         <property name="toolTip">
          <string>Restore default shortcuts.</string>
         </property>
         <property name="text">
          <string>Default</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="promptLabel">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="helpButton">
     <property name="text">
      <string>Help</string>
     </property>
     <property name="icon">
      <iconset theme="help-contextual">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-close">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
